DFE-Digital/teaching-school-hub-finder
# [2158] Replace terrafile

### Context

Ticket: https://trello.com/c/5jCq2a3p

terrafile is no longer maintained

### Changes proposed in this pull request
- Replace terrafile with a simple git command
- Update staging and sandbox domains
- Fix aks-console and aks-ssh commands

## How to review
- Check new domains in README
- make staging terraform-plan
- make domains-infra-plan
- make staging domains-plan
- make staging aks-ssh
- make staging aks-console
